pipeline-steps/deploy.groovy

```diff
@@ -35,4 +35,62 @@ def deploy_sh(Map args) {
   ) // conditionalStage
 }
 
+def deploy_maas() {
+  common.conditionalStage(
+    stage_name: 'Prepare MaaS Configs',
+    stage: {
+      withCredentials([
+        usernamePassword(
+          credentialsId: "dev_pubcloud_user_key",
+          usernameVariable: "PUBCLOUD_USERNAME",
+          passwordVariable: "PUBCLOUD_APIKEY"
+        ),
+        string(
+          credentialsId: "dev_pubcloud_tennant",
+          variable: "PUBCLOUD_TENANT_ID"
+        )
+      ]){
+        sh """
+          export ADMIN_PWD=\$(sudo grep keystone_auth_admin_password /etc/openstack_deploy/user_osa_secrets.yml  | awk '{print \$2}')
```

Comment:
Can you surround this export with a `set +x/set -x` so the admin password isn't visible in the UI?